.gitignore

```diff
@@ -1,2 +1,3 @@
 *.swp
 .DS_Store
+.idea/**
```

Comment:
There is no need for the `/**` glob/wildcard matching here. Just `.idea` will cause Git to ignore everything under that root.

Reply:
Tried it out but did not work (.idea).